apps/molgenis-viz/src/components/display/FileList.vue

```diff
@@ -40,15 +40,24 @@
   </MessageBox>
   <ul class="file-list" v-else>
     <li class="file" v-for="file in data" :key="file.id">
-      <p class="file-element file-name">{{ file.filename }}</p>
+      <p class="file-element file-name">
+        <span v-if="labelsColumn && Object.hasOwn(file, labelsColumn)">
+          {{ file[labelsColumn as string] }}
+        </span>
+        <span v-else>
+          {{ file[fileColumn].filename }}
+        </span>
+      </p>
       <p class="file-element file-format">
-        {{ file.extension }}
+        {{ file[fileColumn].extension }}
       </p>
       <p class="file-element file-size">
-        {{ Math.round((file.size / 1000) * 100) / 100 }} KB
+        {{ (file[fileColumn].size / Math.pow(1024, 2)).toFixed(2) }} MB
       </p>
-      <a class="file-element file-url" :href="file.url">
-        <span class="visually-hidden">Download {{ file.filename }}</span>
+      <a class="file-element file-url" :href="file[fileColumn].url">
+        <span class="visually-hidden"
+          >Download {{ file[fileColumn].filename }}</span
+        >
```

Comment:
brackets are a bit ugly, some enters and the formatter will do it 'correctly' ;)

Reply:
Strange that the formatter broke the lines this way! I manually adjusted the break points.

Reply:
Ye, it does this a bit too often to my taste, those broken tags freak me out 😅